inf/src/main/java/juice/modules/ServerModule.java

```diff
@@ -0,0 +1,35 @@
+package juice.modules;
+
+import config.ConfigFileFinder;
+import config.ConfigurationFactory;
+import config.ServerConfiguration;
+import com.google.inject.AbstractModule;
+import io.logz.guice.jersey.JerseyModule;
+import io.logz.guice.jersey.configuration.JerseyConfiguration;
+
+/**
+ * installs jersey dependencies
+ */
+public class ServerModule extends AbstractModule {
+
+    private final String CONFIG_FILE_NAME = "server.config";
+
+    private JerseyConfiguration jc;
```

Comment:
Let's use full names (`jerseyConfiguration`). We have enough memory and screen inches to deal with longer than 2 char variable names

Reply:
done